.circleci/config.yml

```diff
@@ -0,0 +1,173 @@
+version: 2
+
+jobs:
+
+  test:
+    docker:
+      - image: rust:latest
+    steps:
+      - checkout
+      # Load cargo target from cache if possible.
+      # Multiple caches are used to increase the chance of a cache hit.
+      - restore_cache:
+          keys:
+            - v1-cargo-cache-test-{{ arch }}-{{ .Branch }}
+            - v1-cargo-cache-test-{{ arch }}
+
+      # Install nightly & wasm
+      - run:
+          name: Install Rust nightly
+          command: rustup update nightly && rustup default nightly
+      - run:
+          name: Add wasm32 target
+          command: rustup target add wasm32-unknown-unknown
+
+      # Install wasm tools
+      - run:
+          name: Install wasm-bindgen-cli
+          command: >
+            curl -OL https://github.com/rustwasm/wasm-bindgen/releases/download/0.2.33/wasm-bindgen-0.2.33-x86_64-unknown-linux-musl.tar.gz
+            && tar xf wasm-bindgen-0.2.33-x86_64-unknown-linux-musl.tar.gz
+            && chmod +x wasm-bindgen-0.2.33-x86_64-unknown-linux-musl/wasm-bindgen
+            && mv wasm-bindgen-0.2.33-x86_64-unknown-linux-musl/wasm-bindgen* $CARGO_HOME/bin/
+      - run:
+          name: Install wasm-pack
+          command: >
+            curl -L https://github.com/rustwasm/wasm-pack/releases/download/v0.6.0/wasm-pack-v0.6.0-x86_64-unknown-linux-musl.tar.gz
+            | tar --strip-components=1 --wildcards -xzf - "*/wasm-pack"
+            && chmod +x wasm-pack
+            && mv wasm-pack $CARGO_HOME/bin/
+
+      # Show versions
+      - run:
+          name: Show versions
+          command: rustc --version && cargo --version && wasm-bindgen --version && wasm-pack --version
+
+      # Install node and browser testing tools
+      - run:
+          name: Install nodejs
+          command: >
+            curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash
+            && source ~/.nvm/nvm.sh
+            && nvm install v10.5
+      - run:
+          name: Install latest firefox
+          command: >
+            apt-get update
+            && apt-get install -y libgtk-3-0 libdbus-glib-1-2
+            && wget -q -O - "https://download.mozilla.org/?product=firefox-latest-ssl&os=linux64&lang=en-US"
+            |  tar xj
+      - run:
+          name: Install geckodriver
+          command: >
+            curl --retry 5 -LO https://github.com/mozilla/geckodriver/releases/download/v0.21.0/geckodriver-v0.21.0-linux64.tar.gz
+            && tar xf geckodriver-v0.21.0-linux64.tar.gz
+
+      # Run tests
+      - run:
+          name: Run all tests
+          command: GECKODRIVER=$(pwd)/geckodriver PATH=$(pwd)/firefox:$PATH ./test.sh
```

Comment:
No longer need the `GECKODRIVER` environment variable since we're using `wasm-pack`